### PR TITLE
add git developer OAuth client Id and secret in header when calling github api

### PR DESCRIPTION
**Pull request by fazlerabby07**

### CODE REVIEW

File: Utils/fetchData.tsx [hunk 1]
```diff
@@ -11,9 +11,12 @@ const GetData = (url: string) => {
             })
     });
 };
-const PostData = (url: string, profileUrl: string) => {
+const PostData = (url: string, profileUrl: string, headers?: any) => {
     return new Promise((resolve) => {
-        fetch(url, { method: 'POST', body: profileUrl })
+        console.log('******');
+        console.log(url, profileUrl, headers);
```
Comment (sifatul) [inline Utils/fetchData.tsx:17]: @fazleteraark please remove console.log

File: pages/api/forward-api.ts [hunk 2]
```diff
@@ -4,7 +4,13 @@ import fetch from 'node-fetch';
 
 export default async function handler(req: NextApiRequest, res: NextApiResponse<any>) {
     const body = req.body;
+    const headers = req.headers;
 
+    console.log('====body, headers====')
+    console.log(body);
+    console.log(headers);
```
Comment (sifatul) [inline pages/api/forward-api.ts:11]: @fazlerabby07 please remove header

**Review by sifatul — CHANGES_REQUESTED** [review]

**Comment by sifatul** [comment]
@fazlerabby07 it looks unsafe as headers will be exposed in browser and header contains secret values. is it the best way to do?

**Comment by fazlerabby07** [comment]
@sifatul fix the headers when we call github api. now we convert the github clinet and secret into base64 string.

**Pull request merged**